src/librustdoc/lib.rs

```diff
@@ -338,6 +338,12 @@ fn opts() -> Vec<RustcOptGroup> {
                        "enable-index-page",
                        "To enable generation of the index page")
         }),
+        unstable("persist-doctests", |o| {
+             o.optopt("",
+                       "persist-doctests",
+                       "Persists the rustdoc test executables",
```

Comment:
```suggestion
                       "Directory to persist doctest executables into",
```

Makes it obvious what the path is for.